packages/apps/project/src/components/pages/SubProjectView/SubProjectView.tsx

```diff
@@ -0,0 +1,138 @@
+ *
+ * Copyright (C) 2024 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, {useCallback, useEffect, useMemo} from 'react';
+import {StyleSheet} from 'react-native';
+import {
+  useSelector,
+  useDispatch,
+  useTranslator,
+  clipboardProvider,
+} from '@axelor/aos-mobile-core';
+import {
+  HeaderContainer,
+  HorizontalRule,
+  ScrollList,
+  Text,
+} from '@axelor/aos-mobile-ui';
+import {ProjectHeader} from '../../molecules';
+import {ProjectCard} from '../../atoms';
+import {searchSubProject} from '../../../features/projectSlice';
+
+const SubProjectView = () => {
+  const dispatch = useDispatch();
+  const I18n = useTranslator();
+
+  const {
+    project,
+    loadingSubProject,
+    moreLoadingSubProject,
+    isListEndSubProject,
+    subProjectList,
+  } = useSelector((state: any) => state.project_project);
+
+  const sliceFunctionData = useMemo(
+    () => ({
+      projectId: project.id,
+    }),
+    [project.id],
+  );
+
+  useEffect(() => {
+    dispatch((searchSubProject as any)(sliceFunctionData));
+  }, [dispatch, sliceFunctionData]);
+
+  const fetchSubProjectAPI = useCallback(
+    (page = 0) => {
+      dispatch((searchSubProject as any)({...sliceFunctionData, page}));
+    },
+    [dispatch, sliceFunctionData],
+  );
+
+  return (
+    <>
+      <HeaderContainer
+        fixedItems={<ProjectHeader />}
+        expandableFilter={false}
+      />
+      {project.parentProject != null && (
+        <>
+          <Text style={styles.title} writingType="details">
+            {I18n.t('Project_ParentProject')}
+          </Text>
+          <ProjectCard
+            onPress={() =>
+              clipboardProvider.copyToClipboard(project?.parentProject?.name)
+            }
+            customer={project?.parentProject?.clientPartner}
+            name={project?.parentProject?.name}
+            code={project?.parentProject?.code}
+            company={project?.parentProject?.company?.name}
+            assignedTo={project?.parentProject?.assignedTo?.fullName}
+            projectStatus={project?.parentProject?.projectStatus}
+            parentProject={project?.parentProject?.parentProject?.fullName}
+            isCopyCard={true}
+          />
+        </>
+      )}
+      {subProjectList != null && subProjectList?.length > 0 && (
+        <>
+          <HorizontalRule style={styles.horizontalRule} />
```

Comment:
Same problem as before, you need to hide if there is no parent project. So, hide it when there is no parent projet or when there is no sub projects. Moreover, add a message (maybe a label), when there is neither